src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/transformer/cas2/ApplicationsTransformer.kt

```diff
@@ -69,6 +71,13 @@ class ApplicationsTransformer(
     return ApplicationStatus.inProgress
   }
 
+  private fun getApplicationOriginFromJpa(summary: Cas2ApplicationSummaryEntity): ApplicationOrigin {
+    return when {
+      summary.applicationOrigin == "bail" -> ApplicationOrigin.bail
+      else -> ApplicationOrigin.court
```

Comment:
I think this should be `else -> ApplicationOrigin.hdc` as at this point it's which service is it coming from HDC or Bail

Reply:
would it be good to differentiate the bail types here also?